etc/pdfsam.profile

```diff
@@ -24,6 +24,7 @@ protocol unix
 seccomp
 shell none
 
+private-bin pdfsam,sh,bash,archlinux-java,grep,awk,dirname,uname,which,sort,find,readlink,expr,ls,java-config
```

Comment:
archlinux-java is merely a tool for managing multiple jvm installs (7, 8, 9), and I forgot to actually add 'java'. It still worked though because java is in /usr/lib.